Dockerfile

```diff
@@ -8,6 +8,33 @@ RUN pip install --prefix=/srv --requirement ./requirements.txt
 
 FROM base AS app
 
+ENV GLIBC_VER=2.31-r0
+RUN apk add --no-cache binutils curl \
+    && curl -sL https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub -o /etc/apk/keys/sgerrand.rsa.pub \
+    && curl -sLO https://github.com/sgerrand/alpine-pkg-glibc/releases/download/${GLIBC_VER}/glibc-${GLIBC_VER}.apk \
+    && curl -sLO https://github.com/sgerrand/alpine-pkg-glibc/releases/download/${GLIBC_VER}/glibc-bin-${GLIBC_VER}.apk \
+    && curl -sLO https://github.com/sgerrand/alpine-pkg-glibc/releases/download/${GLIBC_VER}/glibc-i18n-${GLIBC_VER}.apk \
+    && apk add --no-cache \
+        glibc-${GLIBC_VER}.apk \
+        glibc-bin-${GLIBC_VER}.apk \
+        glibc-i18n-${GLIBC_VER}.apk \
+    && /usr/glibc-compat/bin/localedef -i en_US -f UTF-8 en_US.UTF-8 \
+    && curl -sL https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip \
+    && unzip awscliv2.zip \
+    && aws/install \
+    && rm -rf \
+        awscliv2.zip \
+        aws \
+        /usr/local/aws-cli/v2/current/dist/aws_completer \
+        /usr/local/aws-cli/v2/current/dist/awscli/data/ac.index \
+        /usr/local/aws-cli/v2/current/dist/awscli/examples \
+        glibc-*.apk \
+    && find /usr/local/aws-cli/v2/current/dist/awscli/botocore/data -name examples-1.json -delete \
+    && apk --no-cache del \
+        binutils \
+        curl \
+    && rm -rf /var/cache/apk/*
+
```

Comment:
could this be moved to a dedicated target to clean up the result layers?

Reply:
This one is proving to be a bit non-trivial as it's glibc we're trying to copy in

Reply:
< insert crying meme >

Reply:
Looks like the awscliv1 would be enough for this - alternatively, instead of trying to shoehorn glibc into an alpine-based image, if we really really really had to, we should just change the base image.

Reply:
cliv1 worked just fine :)